src/test/httpserver_tests.cpp

```diff
@@ -5,34 +5,52 @@
 #include <httpserver.h>
 #include <test/util/setup_common.h>
 
+#include <event2/http.h>
+
 #include <boost/test/unit_test.hpp>
 
 BOOST_FIXTURE_TEST_SUITE(httpserver_tests, BasicTestingSetup)
 
-BOOST_AUTO_TEST_CASE(test_query_parameters)
+std::optional<std::string> GetParameterFromURI(const std::string_view uri, const std::string& key){
```

Comment:
Missing `#include <string>` and `#include <string_view>`

Reply:
Ok, thanks.